JS/script.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-unused-vars
+function hello() {
```

Comment:
- [Optional] Please give a function name that describes what the function does, eg `toggleMobileMenu` as `hello` is not descriptive.

Reply:
Solved